packages/stateful/components/profile/ProfileProposalCard.tsx

```diff
@@ -108,24 +104,18 @@ export const ProfileProposalCard = ({
     updateProfileName,
   }
 
-  return canVote ? (
+  return couldVote ? (
     <ProfileVoteCard
-      currentVote={vote}
-      currentVoteDisplay={
-        // Fallback to pending since they can vote.
-        <ProposalWalletVote fallback="pending" vote={vote} />
-      }
-      loading={castingVote}
-      onCastVote={castVote}
-      options={options}
-      {...commonProps}
-    />
-  ) : couldVote ? (
-    <ProfileVotedCard
       {...commonProps}
       vote={
-        // Fallback to none since they can no longer vote.
-        <ProposalWalletVote fallback="none" vote={vote} />
+        <ProposalWalletVote
+          fallback={
+            // If they can vote, fallback to pending to indicate that they still
+            // have to vote.
+            canVote ? 'pending' : 'none'
```

Comment:
I find `'none'` (vs `None`) a little sketch. I assume it needs to be a string because it's a prop. Maybe something like `'haveNotVoted'` would be more descriptive/less easily confused for a stringified `None`.